packages/api/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do you use this task? If not remove it.

Reply:
It is code completion

Reply:
It is not. It's a launching task. Remove it.

Reply:
It is not. Remove the .vscode folder under api dir.

Reply:
Done